polkadot-parachains/integritee-runtime/src/weights/pallet_multisig.rs

```diff
@@ -45,18 +45,6 @@ impl<T: frame_system::Config> pallet_multisig::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().writes(1))
```

Comment:
All the weight files are copied and pasted from the kusama runtime. We need to run the benchmarks again before deployment.